README.md

```diff
@@ -1,23 +1,21 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# My Portfolio
 
-> One paragraph statement about the project.
+> This is the first project of my portfolie. It has no functionally as yet.
```

Comment:
- [ ] The previous reviewer pointed out to check the text you are writing for grammar and lexical errors. However, the grammatical error is still there. Please consider replacing `portfolie` with `portfolio`. This will make your readme documentation look professional with no grammatical errors.